pkg/inspector/image-inspector.go

```diff
@@ -133,15 +142,48 @@ func (i *defaultImageInspector) Inspect() error {
 		} else {
 			i.meta.OpenSCAP.Status = iiapi.StatusSuccess
 		}
+		scanResults.Results = append(scanResults.Results, openscap.ParseResults(scanReport)...)
 	}
 
 	if i.imageServer != nil {
 		return i.imageServer.ServeImage(&i.meta,
 			scanReport, htmlScanReport)
 	}
+
+	if len(i.opts.PostResultURL) > 0 {
```

Comment:
Can we move this to a function call just for clarity? This function is already too big